nap/docker/ckan/ckan-plugins/ckanext-napote_theme/ckanext/napote_theme/i18n/fi/LC_MESSAGES/ckanext-napote_theme.po

```diff
@@ -0,0 +1,313 @@
+
+#: ckanext/napote_theme/plugin.py:118 ckanext/napote_theme/plugin.py:129
+#: ckanext/napote_theme/templates/package/snippets/package_basic_fields.html:11
+msgid "Operation Area"
+msgstr "Toiminta-alue"
+
+#: ckanext/napote_theme/plugin.py:119 ckanext/napote_theme/plugin.py:130
+msgid "Tags"
+msgstr ""
+
+#: ckanext/napote_theme/plugin.py:120 ckanext/napote_theme/plugin.py:131
+msgid "Formats"
+msgstr ""
+
+#: ckanext/napote_theme/plugin.py:121 ckanext/napote_theme/plugin.py:132
+msgid "Licenses"
+msgstr ""
+
+#: ckanext/napote_theme/plugin.py:191
+msgid "Transport Service View"
+msgstr "Liikkumispalvelunäkymä"
+
+#: ckanext/napote_theme/templates/base.html:3
+msgid "Data and Resources"
+msgstr "Rajapinnat"
+
+#: ckanext/napote_theme/templates/footer.html:19
+msgid "CKAN API"
+msgstr ""
+
+#: ckanext/napote_theme/templates/footer.html:20
+msgid "CKAN Association"
+msgstr ""
+
+#: ckanext/napote_theme/templates/footer.html:30
+msgid ""
+"<strong>Powered by</strong> <a class=\"hide-text ckan-footer-logo\" "
+"href=\"http://ckan.org\">CKAN</a>"
+msgstr ""
+
+#: ckanext/napote_theme/templates/header.html:36
+msgid "Home"
+msgstr "Etusivu"
+
+#: ckanext/napote_theme/templates/header.html:36
+msgid "Datasets"
+msgstr "Palvelukatalogi"
+
+#: ckanext/napote_theme/templates/header.html:51
+#: ckanext/napote_theme/templates/header.html:60
+#: ckanext/napote_theme/templates/header.html:61
+msgid "View profile"
+msgstr ""
+
+#: ckanext/napote_theme/templates/header.html:67
+msgid "Sysadmin settings"
+msgstr ""
+
+#: ckanext/napote_theme/templates/header.html:69
+msgid "Admin"
+msgstr ""
+
+#: ckanext/napote_theme/templates/header.html:75
+#, python-format
+msgid "Dashboard (%(num)d new item)"
+msgid_plural "Dashboard (%(num)d new items)"
+msgstr[0] ""
+msgstr[1] ""
+
+#: ckanext/napote_theme/templates/header.html:79
+msgid "Dashboard"
+msgstr ""
+
+#: ckanext/napote_theme/templates/header.html:86
+msgid "Edit settings"
+msgstr ""
+
+#: ckanext/napote_theme/templates/header.html:88
+msgid "Settings"
+msgstr ""
+
+#: ckanext/napote_theme/templates/header.html:95
+#: ckanext/napote_theme/templates/header.html:97
+msgid "NAP-data management"
+msgstr "NAP-tietojen hallinta"
+
+#: ckanext/napote_theme/templates/header.html:104
+#: ckanext/napote_theme/templates/header.html:106
+msgid "Log out"
+msgstr ""
+
+#: ckanext/napote_theme/templates/header.html:116
+#: ckanext/napote_theme/templates/home/layout1.html:68
+msgid "Log in"
+msgstr ""
+
+#: ckanext/napote_theme/templates/header.html:125
+#: ckanext/napote_theme/templates/home/layout1.html:74
+msgid "Register"
+msgstr ""
+
+#: ckanext/napote_theme/templates/translations.html:1
+msgid "Test"
+msgstr ""
+
+#: ckanext/napote_theme/templates/home/layout1.html:6
+#, python-format
+msgid "User greeting %(user)s"
+msgstr "Hei %(user)s"
+
+#: ckanext/napote_theme/templates/home/layout1.html:7
+msgid "Datasets-page short info"
+msgstr ""
+"<b>Tietoaineistot</b>-sivulla voit tarkastella ja hakea palvelussa "
```

Comment:
Datasets is translated to Palvelukatalogi previously. Should it be here as well?

Reply:
Correct. "Tietoaineistot" should be "palvelukatalogi". I will fix this.